src/dev/ci_setup/setup.sh

```diff
@@ -30,13 +30,26 @@ fi
 ###
 ### download node
 ###
+UNAME=`uname`
```

Comment:
Can we stick with the `"$(uname)"` syntax? Shellcheck recommends it, and I assume for a good reason.